IlmBase/CMakeLists.txt

```diff
@@ -23,6 +23,8 @@ include(config/ParseConfigure.cmake)
 #set up our project
 project(IlmBase VERSION ${ILMBASE_VERSION} LANGUAGES C CXX)
 
+find_package(Imath)
```

Comment:
Add REQUIRED keyword?

Reply:
I've committed changes that makes this obsolete: if it cmake fails to find Imath, Imath will install it automatically.